lib/routes/account.js

```diff
@@ -567,6 +569,20 @@ module.exports = function (
                       events: events.length
                     })
                   }
+
+                  // IP Profiling sets a flag to bypass sign-in confirmation if this
+                  // user has verified a session with-in the past day from this ip address.
+                  if (ipProfilingEnabled && verified && recency === 'day') {
+                    hasVerifiedSessionFromIp = true
+
+                    log.info({
+                      op: 'Account.ipprofiling.seenAddress',
+                      uid: emailRecord.uid.toString('hex')
+                    })
+
+                  } else {
+                    hasVerifiedSessionFromIp = false
```

Comment:
At https://github.com/mozilla/fxa-auth-server/pull/1525/files#diff-ab87c506e682e952973178a41792df12R566 we log if this IP has not been verified before.